# generated by patch-package 6.4.11
#
# command:
#   npx patch-package a
#
# declared package:
#   a: file:/home/user/src/javascript/patch-package/patch-package-with-pnpm-support/test/packages/a-1.0.0.tgz
#
diff --git a/node_modules/a/index.js b/node_modules/a/index.js
index 7811465..e02f9b1 100644
--- a/node_modules/a/index.js
+++ b/node_modules/a/index.js
@@ -1,6 +1,6 @@
 module.exports = {
   a: 1,
-  b: 2,
+  b: 20,
   c: 3,
   d: 4,
 };
